docs/guides/index.markdown

```diff
@@ -3,22 +3,36 @@ layout: basic
 title: Guides
 ---
 
-### Guides
+## Guides
 
-#### User Guides
+### User Guides
 
 * [AeroGear Android User Guide](aerogear-android)
 * [AeroGear iOS 1.X User Guide](aerogear-ios)
 * [AeroGear iOS 2.X User Guide](aerogear-ios-2.X)
-* [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
-* [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
 
-#### UnifiedPush Server
+### UnifiedPush Server
 
 * [AeroGear UnifiedPush Server resources](../unifiedpush)
 
+### Security
 
-#### Contributing to AeroGear
+* OTP
+  * [AeroGear Two-Factor Authentication with OTP](AeroGear-OTP)
+  * [AeroGear Two-Factor Authentication with OTP as Cordova Plugin](aerogear-cordova/AerogearCordovaOTP)
+
+* OAuth2
+  1. [Basic steps](security/oauth2-basic-steps)
+  2. Client libraries
+
+     The following SDKs were crafted with the simplicity in mind and are constantly improving at each release.
```

Comment:
@lfryc I'm totally cool with it. I was wondering about craft the whole content at the guides section and later move to the landing page. Not sure what do you think, but move only few items, may look odd. Like a blank page with few items.

But I can do that for sure if you think it's better.